app/views/studies/_initialize_metadata_form.html.erb

```diff
@@ -22,13 +22,17 @@
   </div>
   <div class="form-group row">
 		<div class="col-sm-5">
-      <%= f.label :use_metadata_convention, "Do you use SCP conventional names for required metadata column headers?" %> <%= render partial: 'metadata_convention_help_popover', locals: {id: f.object.id.to_s} %><br />
-      <%= f.label :use_metadata_convention_true do %>
+      <%= f.label :use_metadata_convention do %>
+        Do you use SCP conventional names for required metadata column headers?
+        <%= render partial: 'metadata_convention_help_popover', locals: {id: f.object.id.to_s} %>
+      <% end %>
+      <br />
+      <%= f.label :use_metadata_convention_true, 'data-analytics-name': 'metadata-convention-optin' do %>
         <%= f.radio_button :use_metadata_convention, true, checked: true, disabled: !study_file.new_record?%>
         Yes
       <% end %>
       &nbsp;
-      <%= f.label :use_metadata_convention_false do %>
+      <%= f.label :use_metadata_convention_false, 'data-analytics-name': 'metadata-convention-optout' do %>
```

Comment:
Good call logging from the client-side, too.  When convention becomes require for upload via UI, the unconventional uploads will essentially disappear from logs sent via `file_parse_service.rb` and `ingest_job.rb`.  

But this will log users who click "No", see "Convention required, contact us for help", and abandon upload.  That's the important cohort.

Given its importance, please add `studyAccession` to the Mixpanel props sent in logs from the upload wizard.  Currently, it's unavailable; the closest we have is study ID embedded in `appFullPath`, e.g. `/single_cell/studies/5f836edeb067340dbce3a79e/upload`.

Adding `studyAccession` to props sent from upload wizard would let analysts link upload UI events to other events in the same study.

Reply:
oh, I misunderstood the ticket.  I thought this was temporary instrumentation until we removed this option.  I didn't realize we were going to keep the option apparently selectable, even after it was forbidden.  That seems like a dark pattern that will make people angry "you thought you had an easier option, but you were wrong! And now we're going to contact you for even thinking about going the easy route"   I wonder if there's another way to instrument that -- sending events when someone goes to the tab and then navigates away without uploading?  Sending special email alerts when someone uploads a metadata file that is clearly not attempting to be convention compliant?

Reply:
The UI and analytics for SCP-2890 should optimize for learning from that important cohort, the "silent decliners".  These are users who don't upload because they suspect their data isn't conventional, and also don't have strong enough incentives to contact us.  The ticket assumes they'll represent a large proportion of study creators.

You're right a simple "No" option is a bit crude.  At the same time, the update's merely disabled "No" option gives no [affordance](https://en.wikipedia.org/wiki/Affordance) that information is available on hover.  

To address both of those problems, what do you think about a clearer, non-disabled option -- "No, this an issue for my study and I could use help"?  Its obvious clickability would give users an intuitive way to find important information -- "Please contact us at [scp-support@broadinstitute.zendesk.com](mailto:scp-support@broadinstitute.zendesk.com) so we can help or make any updates to the convention" -- while also not confusing them that uploading unconventional metadata is an (easy UI) option.

It'd also preserve the essence of the business goal: giving us a way to reach out to silent decliners, if we desire.  The clearer wording also lets us select for silent decliners who indicate they want help.